src/components/login.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Login = styled.div`
+    margin: 20px;
+    padding: 20px;
+    h2 {
+        font-weight: bold;
+        color: #4A494A;
+    }
+    form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        div {
+            margin: 5px;
+        }
+    }
+`;
+
+class login extends Component {
+    state = {
+        username: '',
+        password: ''
+    }
+
+    handleOnChange = e => this.setState({ [e.target.name]: e.target.value})
+    Login = e => {
+        e.preventDefault();
+
+        axios.post('http://localhost:3000/login', this.state)
+        .then(res => {

```

Comment:
@honda0306 Would you be ok with refactoring this to async / await? 
If yes, do you need help?